app/routes/browse.js

```diff
@@ -12,7 +12,7 @@ export default Ember.Route.extend({
     model() {
         return Ember.RSVP.hash({
             preprints: this.store.findAll('preprint'),
-            taxonomy: this.store.find('taxonomy', 1)
+            taxonomy: this.store.findAll('taxonomy')
```

Comment:
Good point, and Terron will be tackling this soon. 

Realistically we need to refactor this route once real data comes in; we should avoid RSVP.hash in the model hook as much as possible, and we'll need to filter by query params. The current changes were the minimum needed to make the page load at all.